RInAppMessaging/Classes/CampaignDispatcher.swift

```diff
@@ -35,6 +36,15 @@ internal class CampaignDispatcher: CampaignDispatcherType, TaskSchedulable {
         self.router = router
         self.permissionService = permissionService
         self.campaignRepository = campaignRepository
+
+        let sessionConfig = URLSessionConfiguration.default
+        sessionConfig.timeoutIntervalForRequest = Constants.CampaignMessage.imageResourceRequestTimeoutSeconds
+        sessionConfig.urlCache = URLCache(
+            memoryCapacity: 512_000,
+            // response must be <= 5% in order to be cached
+            diskCapacity: 400_000_000, // allocation: 5MB * 4 images / 5%
```

Comment:
True, 400MB is a lot but I don't think it will be used. Maybe instead of increasing disk capacity we could rely on increased memory capacity and bypass that 5% check by setting cache policy to `.returnCacheDataElseLoad` (I'm not sure if that's going to work though)

Reply:
Ok, but I'd rather stick to the suggested tens of MB guideline, and set maybe 40MB or 50MB.
Will that not be sufficient?

If you agree, please make this change in a follow-up PR @apparition47